src/FrameSemantics.cc

```diff
@@ -33,9 +33,11 @@
 #include "sdf/Model.hh"
 #include "sdf/Types.hh"
 #include "sdf/World.hh"
+#include "sdf/Assert.hh"
```

Comment:
Removed the header since it's unused. I must have added it for debugging. https://github.com/ignitionrobotics/sdformat/pull/768/commits/74f06b7d4ce51d0597bcc43b68ba482d1dbd2d97